### ISSUE TITLE
unsafe.js doesn't handle errors on sending

### ISSUE THREAD

**Issue by rlaveycal** [eris-ltd/eris-db.js#31]
```javascript
    var fixedCallback = function(error, data2){
        data2 = data2[1];
        return callback(error, data2);
    };
    this._client.send(rpc.methodName("transactAndHold"), param, fixedCallback);
```

There is no check that data2 is actually defined, so in the event of an error the above throws an exception
```
TypeError: Cannot read property '1' of undefined
    at fixedCallback (/contracts/node_modules/eris-db/lib/unsafe.js:116:22)
```

It needs a guard test
```javascript
    var fixedCallback = function(error, data2){
        if(data2)
            data2 = data2[1];
        return callback(error, data2);
    };
```

**Comment by slowtokyo** [comment]
This overlaps #29 I think.

**Comment by NodeGuy** [comment]
Fixed in 1416aaad6a91680d45b55a20b1324567897f15d5.
